sktime/transformers/pandas_to_numpy.py

```diff
@@ -0,0 +1,18 @@
+from sktime.transformers.base import BaseTransformer
+
+from sktime.utils.transformations import tabularise
+import pandas as pd
+
+class PandasToNumpy(BaseTransformer):
```

Comment:
Do we need a transformer for this one? Transformers should return dataframes to be compatible in pipelines, if you really need a transformer, I'd suggest to make it a helper class (possibly hidden) and to put it into the utils submodule

Reply:
> * mixed with prox forest additions
That's residual from the workshop, these two became intertwined due to distance measures. Once PF PR has been approved I'll merge dev into kernels and it should straighten out.
> * requires some cleaning/reorganisation before merging
Yes, I've maintained this is the case for a while. It's a sizeable job which I'm not prepared to do right now, hence why it's going into contrib (at least that was the idea). I'll sort it out as and when but not sure whether the PR should be approved to get it into contrib (as @TonyBagnall wanted) or to wait until it's polished
> * if you still want to add it now, please move files into contrib submodule
I've now moved everything to do with kernels into the contrib folder

Reply:
I don't know tbh, kernels won't work with dataframes...